go/private/sdk.bzl

```diff
@@ -317,11 +317,17 @@ def _detect_sdk_version(ctx, goroot):
     if result.return_code != 0:
         fail("Could not detect SDK version: '%s version' exited with exit code %d" % (go_binary_path, result.return_code))
 
-    # go version output is of the form "go version go1.18.3 linux/amd64"
+    # go version output is of the form "go version go1.18.3 linux/amd64" or "go
+    # version devel go1.19-fd1b5904ae Tue Mar 22 21:38:10 2022 +0000
+    # linux/amd64". Read the third word, or the fourth word if the third word is
+    # "devel" to find the version number.
```

Comment:
Given that the logic got more complex, can we cite these authorative sources in the comment?
https://github.com/golang/go/blob/2bdb5c57f1efcbddab536028d053798e35de6226/src/cmd/go/internal/version/version.go#L75
https://github.com/golang/go/blob/2bdb5c57f1efcbddab536028d053798e35de6226/src/cmd/dist/build.go#L333

Reply:
No problem, I've updated the PR—let me know if that looks good for you!

Reply:
Looks good! Only the CLA remains, let me know if you run into issues with it.

Reply:
Thanks! Sorry, this is my first time contributing so I need to get added to the right group within my company. I'll let you know when that's done